test/e2e/case22_pod_update_image_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright Contributors to the Open Cluster Management project
+
+package e2e
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"open-cluster-management.io/config-policy-controller/test/utils"
+)
+
+var _ = FDescribe("Test an alternative kubeconfig for policy evaluation", Ordered, func() {
```

Comment:
Looks like a leftover from testing?

```suggestion
var _ = Describe("Test an alternative kubeconfig for policy evaluation", Ordered, func() {
```

Reply:
Thanks for catching that! That's what happens when you try to get a PR up right before the long weekend.  😆